tools/kmod/sof_insert.sh

```diff
@@ -23,6 +23,7 @@ insert_module snd_soc_rt298
 insert_module snd_soc_rt5640
 insert_module snd_soc_rt5645
 insert_module snd_soc_rt5651
+insert_module snd_soc_rt5660
```

Comment:
I have an open question unrelate to this PR: Should we manually load the codec driver? Codec driver should be loaded automatically. If we manually load the codec module, will it hide the potential issue that codec driver is not loaded automatically?

Reply:
@libinyang Before I joined team, there are some codec driver modules exist in the sof_insert.sh. I guess some codec drivers may not be automatically loaded.

Reply:
@libinyang Pierre once said: we need the I2c devices to be probed before the SOF driver, so we add them all in sof_insert. this should explain why they are in sof_insert.sh

Reply:
Right, without reload i2c driver, the i2c devices will fail to load codec driver. This is an i2c driver scope, not our SOF driver scope. This is different from hdmi codec driver. I saw SOF will try to request HDMI codec driver. ASoC codec driver doesn't support this feature. OK, without this feature, we have to manually load the codec driver.